Glyssen/MainForm.cs

```diff
@@ -172,6 +187,20 @@ private void MainForm_Load(object sender, EventArgs e)
 				LoadProject(Settings.Default.CurrentProject);		
 		}
 
+		private void InitializeProgress()
+		{
+			ResetUi();
+			m_tableLayoutPanel.Enabled = false;
+			Cursor = Cursors.WaitCursor;
```

Comment:
It appears this should be 
Cursor.Current = Cursors.WaitCursor;

The way it is now, it seems the UI still never updates with the correct cursor.